ui/src/updateChecker.ts

```diff
@@ -0,0 +1,197 @@
+import * as svelteStore from "svelte/store";
+import persistenStore from "svelte-persistent-store/dist";
```

Comment:
Is this a typo? `persistenStore` -> `persistentStore`.

Reply:
Fixed in 5e26fcf738cf918b39014a3af853fb5be194490c